jupyter_platform/cubeviz_leaflet_prototype_1.ipynb

```diff
@@ -157,43 +145,40 @@
     "\n",
     "app = flask.Flask(__name__)\n",
     "\n",
-    "   \n",
-    "# TODO: Something is not quite right, as this results in weird\n",
-    "# tiling behavior in the viewer.\n",
+    "\n",
+    "# TODO: Need to document this. Tiling still weird.\n",
```

Comment:
@eteq , I copied over this function from your other notebook, still weird when I zoom in/out a few times.